content/docs/command-reference/exp/run.md

```diff
@@ -227,4 +229,8 @@ appending or removing <abbr>parameters</abbr>, make sure to update the
 [`params` section](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#parameters)
 of your `dvc.yaml` accordingly.
 
+Alternatively, you can track all the parameters in the file being modified.
```

Comment:
I think showing an example snippet of `dvc.yaml` in the above example would help.